apps/paper/pages/introduction/comparison.mdx

```diff
@@ -1,10 +1,46 @@
 # Comparison
 
-|                       | [Light](https://light.so) | [NEAR](https://near.org) | [Particle Network](https://particle.network) |
-| --------------------- | :------------------------ | :----------------------- | :------------------------------------------- |
-| Signature Aggregation | ✅                        | 🛑                       | 🛑                                           |
-| EIP-4337 Support      | ✅                        | 🔴                       | ✅                                           |
-| Keystore Sync         | ✅                        | 🔶                       | 🔶                                           |
-| No Trust Assumption   | ✅                        | 🔶                       | 🔶                                           |
-| Signature Aggregation | ✅                        | 🛑                       | 🛑                                           |
-| Multi VMs             | 🛑                        | ✅                       | ✅                                           |
+## Overview with other chain abstraction solutions
+
+|                        | [Light](https://light.so) | [NEAR](https://near.org) | [Particle Network](https://particle.network) |
+| ---------------------- | :------------------------ | :----------------------- | :------------------------------------------- |
+| Signature Aggregation  | ✅                        | 🛑                       | 🛑                                           |
+| EIP-4337 Support       | ✅                        | 🔴                       | ✅                                           |
+| Keystore Sync          | ✅                        | 🔶                       | 🔶                                           |
+| No Trust Assumption    | ✅                        | 🔶                       | 🔶                                           |
+| Programmatic Execution | ✅                        | 🛑                       | 🛑                                           |
+| Multi VMs              | 🛑                        | ✅                       | ✅                                           |
```

Comment:
Suggestion: Alt VMs cuz we do support multi VMs (it just that it has to be EVMs)

Reply:
Yes!! That's such a good point - thank you